notebooks/classification/mcmc_bnn_makemoons.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    plot_binary_class(X_scatters=X,y_scatters=y_hat,XX1_grid=XX1,XX2_grid = XX2,grid_preds_mean=Z,grid_preds_sigma= sigma_Z,titles=("Mean of Predictions of 100  MCMC states","Variance of Predictions of 100  MCMC states"))
</pre><p>Perhaps there are other colormaps that we can use for variance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nipunbatra/bnn-zoo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>